### PR TITLE
webrtc-sys: re-extract boringssl symbols

### PR DESCRIPTION
**Pull request by nbsp**
the previous list was from the bssl version m125 uses, which caused conflicts when we moved to using gclient to load m114.

this one greps for `OPENSSL_EXPORT` in the boringssl code.

### CODE REVIEW

File: webrtc-sys/libwebrtc/boringssl_prefix_symbols.txt [hunk 1]
```diff
@@ -3441,196 +2693,45 @@ i2s_ASN1_OCTET_STRING LK_i2s_ASN1_OCTET_STRING
...
-spx_wots_sign LK_spx_wots_sign
-spx_xmss_pk_from_sig LK_spx_xmss_pk_from_sig
-spx_xmss_sign LK_spx_xmss_sign
+sk_zero LK_sk_zero
+ssl_cert_check_key_usage LK_ssl_cert_check_key_usage
+ssl_client_hello_init LK_ssl_client_hello_init
+ssl_decode_client_hello_inner LK_ssl_decode_client_hello_inner
+ssl_is_valid_ech_public_name LK_ssl_is_valid_ech_public_name
+ssl_session_serialize LK_ssl_session_serialize
+uint16_t LK_uint16_t
```
Comment (theomonnom) [inline webrtc-sys/libwebrtc/boringssl_prefix_symbols.txt]: this doesn't seems right? how are you extracting the types?

**Review by theomonnom — APPROVED** [review]
otherwise lg

File: webrtc-sys/libwebrtc/boringssl_prefix_symbols.txt (hunk 1 above)
Comment (nbsp) [reply webrtc-sys/libwebrtc/boringssl_prefix_symbols.txt]: oops yeah, this one has a newline in the definition so it got the type. i'll have a manual comb through for those

Comment (theomonnom) [reply webrtc-sys/libwebrtc/boringssl_prefix_symbols.txt]: can you also push the script that get the symbols, so it is reproducible by everyone

Comment (nbsp) [reply webrtc-sys/libwebrtc/boringssl_prefix_symbols.txt]: i'll add it commented out to the top of the symbol rename list

**Pull request merged**